src/clj/api/config.clj

```diff
@@ -54,4 +54,4 @@
      :public-key
      (or (get-env-variable "PUBLIC_KEY")
          "-----BEGIN PUBLIC KEY-----\nMFwwDQYJKoZIhvcNAQEBBQADSwAwSAJBAJliLjOIAqGbnjGBM1RJml/l0MHayaRH\ncgEg00O9wBYvoNXrstFSzKTCKtG5MayUKgdG7C/98nu/TEzhvRFjINcCAwEAAQ==\n-----END PUBLIC KEY-----\n")
-     :private-key (or (get-env-variable "PRIVATE_KEY") private-key)}))
+     :private-key (or private-key (get-env-variable "PRIVATE_KEY"))}))
```

Comment:
why revert that order?

Reply:
Sorry I wasn't suppose to commit it, will remove it. On my dev env it wasn't finding the PK so I had to revert that to make it work

Reply:
weird it would not find it, did you export the env var?